draft-ietf-tls-external-psk-guidance.md

```diff
@@ -154,12 +166,14 @@ participating in a connection.
 
 # PSK Security Properties {#sec-properties}
 
-External PSK authentication in TLS allows endpoints to authenticate connections
-using previously established keys. These keys do not provide protection
-of endpoint identities (see {{endpoint-privacy}}), nor do they provide
+When using external PSK authentication, the use of previously established
+keys allows TLS endpoints to authenticate the endpoint identities.  However,
+these keys do not provide privacy protection of endpoint identities
+(see {{endpoint-privacy}}), nor do they provide
 non-repudiation (one endpoint in a connection can deny the conversation).
 Protection of endpoint identities and protection against an endpoint denying
```

Comment:
I would recommend deleting this entire sentence, for two reasons: (1) the server identity is still leaked even if certificates are used (and ECH is not used), and (2) there may be other future forms of authentication that prohibit deniability. It seems reasonable to just say "PSK-based auth has these properties," and nothing more.

Reply:
I think you are suggesting:

When using external PSK authentication, these TLS 1.3 does not provide privacy protection ...

If that is what you mean, then I a comfortable.
If you mean something else, please provide words.

Reply:
Yeah, that's what I mean.